benchmarks/llm/perf.sh

```
@@ -14,42 +14,103 @@
+        --osl) osl="$2"; shift ;;
+        --url) url="$2"; shift ;;
+        --concurrency) IFS=',' read -r -a concurrency_array <<< "$2"; shift ;;
+        --help) print_help; exit 0 ;;
+        *) echo "Unknown parameter passed: $1"; print_help; exit 1 ;;
+    esac
+    shift
+done
+
+# If concurrency_array is not set by user, use default
+if [ -z "${concurrency_array+x}" ]; then
+    concurrency_array=($concurrency_list)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)